backend/docs/interview_schema.md

```diff
@@ -0,0 +1,248 @@
+# Interview Schema Backend Design
+
+**INTERVIEW**
+
+|   id  |   candidate_id  |   interviewer_key    |   sections  |  overall_score  |  general_notes  |
+|:------:|:-------:|:------:|:-------:|:------:|:-------:|
+
+**SECTION**
+
+|   id  |  description | questions | section_notes |
+|:------:|:-------:|:-------:|:-------:|
+
+**QUESTION**
+
+|   id  |   score   | question_text |
+|:------:|:-------:|:------:|
+
+
+## Endpoints Documentation 
+
+
+GET ALL INTERVIEWS: 
+* GET /interviews | interview
```

Comment:
This can just be `GET /`. Also is the stuff after the `|` what each endpoint returns?